src/utilstrencodings.cpp

```diff
@@ -10,6 +10,7 @@
 #include <cstdlib>
 #include <cstring>
 #include <errno.h>
+#include <iomanip>
```

Comment:
Oh,. yes. This is a relict of a custom URIencode/decode implementation. Will remove.